src/main/java/org/beanpod/switchboard/config/WebSecurityConfig.java

```diff
@@ -0,0 +1,45 @@
+package org.beanpod.switchboard.config;
+
+import lombok.AllArgsConstructor;
+import org.beanpod.switchboard.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
+import org.springframework.security.config.annotation.web.builders.WebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+
+@Configuration
+@AllArgsConstructor
+public class WebSecurityConfig extends WebSecurityConfigurerAdapter {
+
+  private final UserService userService;
+
+  private final BCryptPasswordEncoder bCryptPasswordEncoder;
```

Comment:
To reduce the code smell related to `Member name 'bCryptPasswordEncoder' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9]*$'`. I'd suggest refactoring `bCryptPasswordEncoder` --> `bcryptPasswordEncoder` or `passwordEncoder`. Any name such that the second character is a lowercase letter, or number would be compliant.

Reply:
fixed